setup.py

```diff
@@ -48,6 +48,7 @@
     "safetensors",
     "pytest < 8.0.0",
     "torchsde",
+    "timm",
```

Comment:
Can you tell a bit more about why timm is needed?

Reply:
Okay, I saw the discussion above

Reply:
But I guess it's needed for the example too no?

Reply:
For the example code, I explicitly set it not to use timm module (i.e. no_timm). So we do not need to import timm for that.
`model = DetrForObjectDetection.from_pretrained("facebook/detr-resnet-101", revision="no_timm")`